pkgs/servers/xmpp/prosody/default.nix

```diff
@@ -47,6 +49,17 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ makeWrapper ];
+
+  # Be sure to update this whenever jitsi-meet is updated.
+  jitsiMeet = if withJitsiModules then
```

Comment:
TOREMOVE

Instead, you can copy the `/resources/prosody-plugins` directory to a specific location (eg. `$out/resources/prosody-plugins`) in the `jitsi-meet` derivation.